src/main/java/pl/project13/maven/git/GitCommitIdMojo.java

```diff
@@ -631,21 +633,26 @@ static Properties readJsonProperties(@NotNull File jsonFile) throws CannotReadFi
       Closeable closeable = null;
 
       try {
-        final FileInputStream fis = new FileInputStream(jsonFile);
-        closeable = fis;
-
-        final InputStreamReader reader = new InputStreamReader(fis, Charsets.UTF_8);
-        closeable = reader;
-
-        final ObjectMapper mapper = new ObjectMapper();
-        final TypeReference<HashMap<String,Object>> mapTypeRef =
-          new TypeReference<HashMap<String,Object>>() {};
-
-        propertiesMap = mapper.readValue(reader, mapTypeRef);
+        boolean threw = true;
+        try {
```

Comment:
The clean solution would be to use try-with-resources, but for that we need Java 7.